activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java

```diff
@@ -76,7 +76,7 @@ public static Builder newBuilder() {
     ActiveMQConnectionFactory connectionFactory;
     String queue = "zipkin";
     Encoding encoding = Encoding.JSON;
-    int messageMaxBytes = 100000; // arbitrary to match kafka, messages theoretically can be 2GiB
+    int messageMaxBytes = 500_000; // arbitrary to match kafka, messages theoretically can be 2GiB
```

Comment:
If I'm not mistaken, this doesn't match Kafka anymore which is 1MB. Or maybe there's something related to bi-directionality here.

Could you explain more why we want to use half of Kafka max size as our default? If the reason is just to be somewhat conservative as a default and let users optimize themselves, it makes sense, but in that case I guess we don't need to mention Kafka in these comments.

Reply:
@anuraaga yes, it's a conservative default as you mention. I will improve docs.